functions/src/aqi-calculation/index.ts

```diff
@@ -1,22 +1,29 @@
 import * as functions from 'firebase-functions';
```

Comment:
index.ts should remain in the src directory in order for the cloud functions to deploy--just helper functions and files can be moved to subfolders